app/controllers/application_controller.rb

```diff
@@ -53,6 +53,8 @@ def ensure_not_mentor!
     redirect_to mentoring_inbox_path
   end
 
+  def country_code = request.location&.country_code.presence
```

Comment:
We need to ensure this is instant and foolproof if going here. That means the database needs to be local to this docker image really, and the lookup in like 1ms.

The alternative, which feels safer to me, is to pass the IP address here into the `Metric::Queue` method, then do the lookup as part of the `Metric::Create` call